[2025-05-07T01:35:24.642+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-07T01:35:24.660+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.generate_dbt_docs manual__2025-05-07T01:02:20.058495+00:00 [queued]>
[2025-05-07T01:35:24.693+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.generate_dbt_docs manual__2025-05-07T01:02:20.058495+00:00 [queued]>
[2025-05-07T01:35:24.703+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 3
[2025-05-07T01:35:24.741+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): generate_dbt_docs> on 2025-05-07 01:02:20.058495+00:00
[2025-05-07T01:35:24.762+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=13045) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-07T01:35:24.764+0000] {standard_task_runner.py:72} INFO - Started process 13060 to run task
[2025-05-07T01:35:24.765+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'generate_dbt_docs', 'manual__2025-05-07T01:02:20.058495+00:00', '--job-id', '760', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmpi8ufjxkb']
[2025-05-07T01:35:24.767+0000] {standard_task_runner.py:105} INFO - Job 760: Subtask generate_dbt_docs
[2025-05-07T01:35:24.820+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.generate_dbt_docs manual__2025-05-07T01:02:20.058495+00:00 [running]> on host 8280c28b7f0f
[2025-05-07T01:35:24.893+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='generate_dbt_docs' AIRFLOW_CTX_EXECUTION_DATE='2025-05-07T01:02:20.058495+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-07T01:02:20.058495+00:00'
[2025-05-07T01:35:24.895+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-07T01:35:24.895+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-07T01:35:24.896+0000] {logging_mixin.py:190} INFO - Current task name:generate_dbt_docs state:running start_date:2025-05-07 01:35:24.660834+00:00
[2025-05-07T01:35:24.897+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-07T01:35:24.898+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-07T01:35:24.902+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-05-07T01:35:24.904+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt_project/bank_reviews_dbt && dbt docs generate']
[2025-05-07T01:35:24.923+0000] {subprocess.py:99} INFO - Output:
[2025-05-07T01:35:27.360+0000] {subprocess.py:106} INFO - [0m01:35:27  Running with dbt=1.9.4
[2025-05-07T01:35:27.732+0000] {subprocess.py:106} INFO - [0m01:35:27  Registered adapter: postgres=1.9.0
[2025-05-07T01:35:28.276+0000] {subprocess.py:106} INFO - [0m01:35:28  Found 5 models, 12 data tests, 433 macros
[2025-05-07T01:35:28.281+0000] {subprocess.py:106} INFO - [0m01:35:28
[2025-05-07T01:35:28.282+0000] {subprocess.py:106} INFO - [0m01:35:28  Concurrency: 1 threads (target='dev')
[2025-05-07T01:35:28.283+0000] {subprocess.py:106} INFO - [0m01:35:28
[2025-05-07T01:35:28.966+0000] {subprocess.py:106} INFO - [0m01:35:28  Building catalog
[2025-05-07T01:35:29.091+0000] {subprocess.py:106} INFO - [0m01:35:29  Catalog written to /opt/***/dbt_project/bank_reviews_dbt/target/catalog.json
[2025-05-07T01:35:30.505+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-05-07T01:35:30.535+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-07T01:35:30.536+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=generate_dbt_docs, run_id=manual__2025-05-07T01:02:20.058495+00:00, execution_date=20250507T010220, start_date=20250507T013524, end_date=20250507T013530
[2025-05-07T01:35:30.563+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-07T01:35:30.564+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-07T01:35:30.564+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-07 01:31:20.613033+00:00
[2025-05-07T01:35:30.565+0000] {logging_mixin.py:190} INFO - Task hostname:8280c28b7f0f operator:BashOperator
[2025-05-07T01:35:30.578+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-07T01:35:30.588+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
